app/controllers/admin/invoices_controller.rb

```diff
@@ -1,5 +1,10 @@
 class Admin::InvoicesController < ApplicationController
   def show
-    
+    @invoice = Invoice.find(params[:id])
+    @invoice_items = @invoice.invoice_items
```

Comment:
I feel like it's minor, but I was dinged on an IC for setting up something like this. In the view, we could just call @invoice.invoice_items and slim up the controller as much as possible..